vite-project-kabinet/src/Content/CardFullArticle.jsx

```diff
@@ -1,26 +1,42 @@
+import { useState } from "react"
 import "./article.style.css"
+import { useEffect } from "react"
 
 export const CardFullArticle = ({title}) => {
+
+    // todo: komponenta CardFullArticle přijme jenom komponentu ID, title načte pomocí fetch
+    const [mojeData, setMojeData] = useState({})
+
+    //todo: fetch(`https://www.casopiskabinet.sk/wp-json/wp/v2/posts/${id}`)
+    const getPromisePostId = (id) => {
```

Comment:
Vidím, že se tu poctivě snažíš o přístup co jsem se Ti snažil vnutit. Ale klidně to uprav a rovnou načítej článek z WordPressu. Pro seznam to už stejně máš.